### Import packages

In [1]:
# import packages
import pandas as pd
import numpy as np

## Task 1

In [2]:
prob_2 = 1/6
prob_3 = 1/6

prob_2_or_3 = prob_2 + prob_3

print(prob_2_or_3)

0.3333333333333333


<p>
<details><summary>Click to show answer</summary><br>
    
`0.3333333333333333`

</details>
</p>

<p>
<details><summary>Click to show solution</summary><br>
    
```python
prob_2 = 1/6
prob_3 = 1/6

prob_2_or_3 = prob_2 + prob_3

print(prob_2_or_3)
```

</details>
</p>

## Task 2

In [3]:
# import dataset, set index to "Breathlessness"
breath_data = pd.read_csv("datasets/breathlessness.csv")
breath_data = breath_data.set_index("Breathlessness")

display(breath_data.head())

Boys  Girls  Total
Breathlessness                    
Yes              231    285    516
No               127     89    516
Total            358    374    732

In [6]:
# calculate the probability of suffering from breathlessness
num_breathless = breath_data.loc['Yes', 'Total']
total_people = breath_data.loc['Total', 'Total']

prob_breathless = num_breathless / total_people
print(prob_breathless)

0.7049180327868853


In [9]:
breath_data.loc['Yes','Girls']

285

<p>
<details><summary>Click to show answer</summary><br>

`0.7049180327868853`

</details>
</p>

<p>
<details><summary>Click to show solution</summary><br>

```python
# calculate the probability of suffering from breathlessness
num_breathless = breath_data.loc['Yes', 'Total']
total_people = breath_data.loc['Total', 'Total']

prob_breathless = num_breathless/total_people
print(prob_breathless)
```

</details>
</p>

## Task 3

In [10]:
display(breath_data.head())

Boys  Girls  Total
Breathlessness                    
Yes              231    285    516
No               127     89    516
Total            358    374    732

In [11]:
# calculate the probability of breathlessness for a participant
# given that this participant is a girl
num_breathless_girls = breath_data.loc['Yes','Girls']
num_girls = breath_data.loc['Total','Girls']

prob_breathless_girl = num_breathless_girls / num_girls

print(prob_breathless_girl)

0.7620320855614974


In [12]:
# calculate the probability of breathlessness for a participant
# given that this participant is a boy
num_breathless_girls = breath_data.loc['Yes','Boys']
num_girls = breath_data.loc['Total','Boys']

prob_breathless_girl = num_breathless_girls / num_girls

print(prob_breathless_girl)


0.6452513966480447


<p>
<details><summary>Click to show answer</summary><br>

`0.7620320855614974`

</details>
</p>

<p>
<details><summary>Click to show solution</summary><br>

```python
# calculate the probability of breathlessness for a participant, given that this participant is a girl
num_breathless_girls = breath_data.loc['Yes', 'Girls']
num_girls = breath_data.loc['Total', 'Girls']

prob_breathless_girl = num_breathless_girls/num_girls

print(prob_breathless_girl)
```

</details>
</p>

## Task 4

In [13]:
# calculate the probability of not breathlessness for a participant
# given that this participant is a boy
num_breathless_girls = breath_data.loc['No','Boys']
num_girls = breath_data.loc['Total','Boys']

prob_breathless_girl = num_breathless_girls / num_girls

print(prob_breathless_girl)

0.3547486033519553


<p>
<details><summary>Click to show answer</summary><br>

`0.6452513966480447`

</details>
</p>

<p>
<details><summary>Click to show solution</summary><br>

```python
# calculate the probability of breathlessness for a participant, given that this participant is a boy
num_breathless_boys = breath_data.loc['Yes', 'Boys']
num_boys = breath_data.loc['Total', 'Boys']

prob_breathless_boy = num_breathless_boys/num_boys

print(prob_breathless_boy)
```

</details>
</p>

## Task 5

In [ ]:
# calculate the probability of TB, given a positive test
prob_b_given_a =
prob_a = 
prob_b = 

prob_tb_postest = 

print(prob_tb_postest)

<p>
<details><summary>Click to show answer</summary><br>

`0.9191757404129792`

</details>
</p>

<p>
<details><summary>Click to show solution</summary><br>

```python
# calculate the probability of TB, given a positive test
prob_b_given_a = 0.99808
prob_a = 1561/2373
prob_b = 1695/2373

prob_tb_postest = (prob_b_given_a*prob_a)/prob_b

print(prob_tb_postest)
```

</details>
</p>